src/MgvOfferMaking.sol

```diff
@@ -52,7 +52,7 @@ contract MgvOfferMaking is MgvHasOffers {
     }
   }
 
-  function newOfferByTick(OLKey memory olKey, int tick, uint gives, uint gasreq, uint gasprice)
+  function newOfferByTick(OLKey memory olKey, Tick tick, uint gives, uint gasreq, uint gasprice)
```

Comment:
Is this going to annoying for callers? Or will it be natural for them to construct a `Tick` using `TickConversionLib`? How about off-chain, eg will ethers.js handle this nicely?

Reply:
ethers.js handles custom types well, and I think for contract writers the exported type safety is worth it; normally you only input literals when writing tests and real contracts use conversion functions.